conduct.md

```diff
@@ -101,7 +101,7 @@ Behavior that will lead to exclusion includes the following points, inspired by
 * Violence, threats of violence or violent language directed against another person.
 * Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
 * Posting or displaying sexually explicit or violent material.
-* Posting or threatening to post other people’s personally identifying information ("doxing").
+* Posting or threatening to post other people’s personally identifying information.
```

Comment:
I think it would help to clarify how this relates to public information

```suggestion
* Posting or threatening to post other people’s (non-public) personally identifying information.
```

Reply:
I like the spirit of the change, but if personal identifying information is public but hard to find, and someone digs it out of an old usenet archive or something and posts it in the context of a heated argument, that is still very bad because it is frightening and potentially endangering them.  Likewise, if someone happens to have a listed telephone number, tracking it down and posting it is bad.  So I'm not 100% sold on this change.  It might admit too much abuse for not enough clarification that sharing e.g. their twitter handle is fine.

Reply:
Why do we want to add wording that suggests that it's okay to 'post or threaten to post' other people’s **public** personally identifying information?

Reply:
@nafg - Well, if I am saying something like, "Hey I really like Li Haoyi's requests web server libraries" and someone else replies, "Oh really?  That sounds good to me, but I have a few tricky requirements. I'd love to talk to him, if you think he'd not mind some questions" then I'm kind of in a bind if I can't mention any identifying information at all.  You couldn't ever refer anyone to anyone if literally no personally identifying information could be posted.